seccomp/src/lib.rs

```diff
@@ -407,10 +376,31 @@ pub struct SeccompRule {
     action: SeccompAction,
 }
 
+/// Type that encapsulates a tuple (syscall number, rule set).
+///
+pub type SyscallRuleSet = (i64, Vec<SeccompRule>);
+
+/// Builds the (syscall, rules) tuple for allowing a syscall regardless of arguments.
+///
+#[inline(always)]
+pub fn allow_syscall(syscall_number: i64) -> SyscallRuleSet {
+    (
+        syscall_number,
+        vec![SeccompRule::new(vec![], SeccompAction::Allow)],
+    )
+}
+
+/// Builds the (syscall, rules) tuple for allowing a syscall with certain arguments.
+///
+#[inline(always)]
+pub fn allow_syscall_if(syscall_number: i64, rules: Vec<SeccompRule>) -> SyscallRuleSet {
+    (syscall_number, rules)
+}
+
 /// Filter context containing rules assigned to syscall numbers.
 pub struct SeccompFilterContext {
-    /// Hash map, mapping a priority and a chain of rules to a syscall number.
-    rules: HashMap<i64, (i64, Vec<SeccompRule>)>,
+    /// Hash map, mapping a chain of rules to a syscall number.
```

Comment:
I know this has been called this since forever, but, since you're changing stuff here, can we call this `SeccompFilter`? I don't get the `Context` part - sounds like a catch-all name, like saying `SeccompFilterStuff`.